django/db/backends/sqlite3/operations.py

```diff
@@ -27,7 +27,7 @@ def bulk_batch_size(self, fields, objs):
         return (limit // len(fields)) if len(fields) > 0 else len(objs)
 
     def check_expression_support(self, expression):
-        bad_fields = (fields.DateField, fields.DateTimeField, fields.TimeField)
+        bad_fields = (fields.DateField, fields.DateTimeField, fields.TimeField, fields.DurationField)
```

Comment:
Sorry, I didn't provide enough context here for how I imagined this to be fixed. The existing logic here enforces that aggregations can't be used with these fields. Aggregating a duration field is fine on sqlite.

Instead, you should write a check here like:
```python
try:
    if isinstance(expression, Extract) and isinstance(expression.output_field, fields.DurationField):
        raise NotImplementedError("Extract .. on DurationField fails on sqlite")
except FieldError:
    pass
```

Does that make sense?